README.md

```diff
@@ -224,9 +224,44 @@ products_with_names_only = session.query({
 |`all`|Returns all documents from result set (considering `take` / `skip` options)|
 |`count`|Returns count of all documents matching query criteria (non-considering `take` / `skip` options)|
 
+## Working with secured server
+1. Put generated client certificate somewhere on your app structure. Optionally you can store it as string on your application config file or in the database
+2. Instantiate `RavenDB::StoreAuthOptions`. Pass path to the certificate file (or string with its contents) and passphrase (if needed) to constructor:
+```ruby
+auth_options = RavenDB::StoreAuthOptions.new('./path/to/certiface.pem')
```

Comment:
No, we don't want to deal with any paths, etc.
User _must_ provide the _string_ of the PEM , with optional password.
How they store it is out of scope for us

Reply:
ok, will change